Jenkinsfile

```diff
@@ -81,6 +81,109 @@ pipeline {
                 }
             }
         }  
+
+        stage('Run Codewind test suite') {
+            
+                steps {
+                    withEnv(["PATH=$PATH:~/.local/bin;NOBUILD=true"]){
+                    withDockerRegistry([url: 'https://index.docker.io/v1/', credentialsId: 'docker.com-bot']) {
+                        sh '''#!/usr/bin/env bash
+                        echo "Starting tests for Eclipse Codewind ..."
+                        export PATH=$PATH:/home/jenkins/.jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/node_js/bin/
+
+                        ARCH=`uname -m`;
+                        printf "\n\n${MAGENTA}Platform: $ARCH ${RESET}\n"
+
+                        # Install nvm to easily set version of node to use
+                        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+                        export NVM_DIR="$HOME/.nvm" 
+                        . $NVM_DIR/nvm.sh
+                        nvm i 10
+
+                        # Install docker-compose 
+                        curl -L https://github.com/docker/compose/releases/download/1.21.2/docker-compose-`uname -s`-`uname -m` -o ~/docker-compose
+                        chmod +x ~/docker-compose
+
+                        # Run eslint on portal code
+                        cd src/pfe/portal
+                        npm install
+                        npm run eslint
+                        if [ $? -ne 0 ]; then
+                            exit 1
+                        fi
+                        cd ../../..
```

Comment:
Is there an environment variable for the codewind directory that we can use to change directory instead of relative paths? If not we can create one before `cd src/pfe/portal`. It'd help make the Jenkinsfile more robust in case other changes are added in between in the future.

Reply:
there isn't right now but I do agree we could improve on this.   Ideally, if we can get this in and working as is, we can iterate over it and make changes